CONTRIBUTING.md

```diff
@@ -24,7 +48,8 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure commit messages are in the proper format (see below).
```

Comment:
This mentions formatting but does not mention anything "below"

Reply:
AFAIK we don't really have hard formatting requirements so I'm just going to remove this bit.